packages/frontend/src/components/RegisterForms/VerifyEmailForm.tsx

```diff
@@ -85,8 +95,33 @@ const VerifyEmailForm: FC<{ isLoading?: boolean }> = ({ isLoading }) => {
                     key={expirationTime?.toString()}
                     date={expirationTime}
                     renderer={({ minutes, seconds, completed }) => {
-                        if (completed || data?.otp?.isMaxAttempts) {
-                            return <></>;
+                        if (completed) {
+                            return (
+                                <Alert
+                                    icon={
+                                        <MantineIcon icon={IconAlertCircle} />
+                                    }
+                                    color="orange.8"
+                                    radius="xs"
+                                >
+                                    Your one-time password expired. Please
+                                    resend a verification email.
+                                </Alert>
+                            );
+                        }
+                        if (data?.otp?.isMaxAttempts) {
+                            return (
+                                <Alert
+                                    icon={
+                                        <MantineIcon icon={IconAlertCircle} />
+                                    }
+                                    color="orange.8"
+                                    radius="xs"
+                                >
+                                    Maximum number of attempts reached. Please
+                                    verification email.
```

Comment:
```suggestion
                                    request another verification email.
```
?

Reply:
Not sure of the wording ^  maybe `Please
                                    resend a verification email.`?

Reply:
What about "Hit 'Resend Email' to receive a new one-time passcode"

Reply:
Makes me think that "resend email" should be "send me another one-time passcode"

Reply:
Maybe one-time passcode is to devvy? Verification code?

Reply:
> "Hit 'Resend Email' to receive a new one-time passcode"

love this one!

Reply:
> Verification code 

this one 🆙

Reply:
![CleanShot 2023-11-16 at 14 47 22@2x](https://github.com/lightdash/lightdash/assets/11660098/fb0736ef-b24d-4e9f-bfe5-f67e700af9ca)

too bold?

Reply:
I like the bold, just making the text smaller would be nicer: 

```tsx
<Alert
                icon={<MantineIcon icon={IconAlertCircle} />}
                color="orange.8"
                radius="xs"
mt='sm'
            >
                <Text fz="xs">
                    Your verification code has expired.{' '}
                    <Text span fw={500}>
                        Resend a verification email{' '}
                    </Text>{' '}
                    to receive a new code
                </Text>
            </Alert>
```

Also, on the above ^ added a `mt` so that the spacing is a bit more levelled(?)